### ISSUE TITLE
Missing "over area Ac"

### ISSUE THREAD

**Issue by elwazana** [smiths/swhs#48]
https://github.com/smiths/swhs/blob/0fa29b3812e325c38c69f9e99aed687a5938380d/SRS/PCM_SRS.tex#L911

See Dr. Smith's comment on issue #33

**Comment by elwazana** [comment]
@smiths Should I continue doing this for the Drasil side for now, or should I wait for @JacquesCarette and @szymczdm response to this issue as we discussed? https://github.com/JacquesCarette/Drasil/issues/578

**Comment by smiths** [comment]
You should wait.

**Comment by JacquesCarette** [comment]
Should we assign this issue to @szymczdm or create a new one for that purpose?

**Comment by smiths** [comment]
I'll start with having a look at how we can remove the need to reference Ac as a note after the equation.  We should be able to change how the equations are derived/combined.  We should be able to make the integration explicit to bring in the Ac this way.  We can discuss this further during the "all hands" meeting tomorrow (June 12).

**Comment by smiths** [comment]
I believe I have good news about the reference to Ac.  We do not need to attach it to the equation, since the equation is about thermal flux.  The thermal flux equation doesn't depend on the area; it acts over an area.  The connection to the area is made through the derivation of GD2 (Simplified rate of change of temperature) and the use of this equation in IM1 and IM2 (Energy balances for water and PCM, respectively).

The changes to the manual version of the SWHS SRS are available at:

https://github.com/smiths/swhs/commit/2106695f9fb6c5d8f5215779bb489b698c17a7ca

 I've indicated the important changes using comments (\wss).

@elwazana, can you please make the changes to the Drasil code for Ac and Ap (and elsewhere) as indicated by the \wss comments?  The DD1 and DD2 cases are simpler than we previously discussed, but the connection to Newton's Law of Cooling still needs to be added.  From discussion with @szymczdm and @JacquesCarette, we should be able to add this information as part of the derivation.  There are other examples of derivations in the documentation.  This derivation will follow that patten, but it will be shorter.

We will discuss this in our meeting this afternoon, but it appears that we will need to fix most of the case studies to add back in the derivation information that was inadvertently removed.

**Comment by elwazana** [comment]
@smiths Hello Dr. Smith, I just need some clarification on what I'm changing? I've been looking back and forth between the \wss comments and the current stable files for Drasil (just as a reference point to where the appropriate code is) and some of the changes proposed in the comments don't even appear in the Stable

Example:
https://github.com/smiths/swhs/blob/2106695f9fb6c5d8f5215779bb489b698c17a7ca/docs/SRS/PCM_SRS.tex#L916

https://github.com/smiths/swhs/blob/2106695f9fb6c5d8f5215779bb489b698c17a7ca/docs/SRS/PCM_SRS.tex#L935-L937
(This paragraph above in its entirety doesn't even appear in the description)

Current Stable:
![image](https://user-images.githubusercontent.com/35268928/41299608-daba5fc2-6e31-11e8-9c7a-2a25f6b7a141.png)

**Comment by smiths** [comment]
@elwazana, remember this entire issue was started by stable being incorrect.  When I wrote remove in SWHS that doesn't mean that it should be removed from stable; it means it should be removed from what stable should have said.  😄 If one of my \wss comments says remove, and it isn't in stable, then you can interpret that as meaning, don't add to stable.

**Comment by elwazana** [comment]
@smiths So I'm assuming then that I should add the rest of this paragraph except the \wss Remove comment?

https://github.com/smiths/swhs/blob/2106695f9fb6c5d8f5215779bb489b698c17a7ca/docs/SRS/PCM_SRS.tex#L935-L937

**Comment by smiths** [comment]
Correct.  Your goal is to make stable (and the generated code) look like the manual version, just with the removed text not included.  The assumption you have highlighted should be something that you can add to the derivation, along with the mention of Newton's Law of Cooling.

**Comment by elwazana** [comment]
@smiths By add to the derivation, do you mean the Instance Model derivation? Because DataDefs don't have a derivation section.

**Comment by smiths** [comment]
We need to add a derivation section to the data definition.  As I alluded to above, @JacquesCarette and @szymczdm and I discussed how to add the removed knowledge back to Drasil.  The description field is now auto-generated, so it doesn't easily fit back into the description.  However, we have the ability to add derivation knowledge.  It isn't currently part of the data definition, but @szymczdm said there is no reason it couldn't be.  Therefore, we can add the derivation to the DD so that we can recapture the deleted knowledge.

We can discuss this further in our meeting this afternoon.

**Comment by smiths** [comment]
@elwazana, I'm afraid that I don't know the answer to your question.  Hopefully it is a straightforward answer for @szymczdm.

**Comment by szymczdm** [comment]
@elwazana I think this is an issue of deprecated code more than anything. Data Defs should be constructed very similarly to IMs (except DDs are `[QDefinition]` instead of `RelationConcept`). I believe this is due to the issues with the Devi branch not being solved yet.

**Comment by smiths** [comment]
In looking through open issues, I can verify that this issue should still be open.  The requested changes to have the manual version of SWHS match the Drasil version have not yet happened.  After a discussion with @JacquesCarette, the problem seems to be that the assumptions should be referenced in the "Notes" field of the chunks, but this is the opposite of the order in how Drasil currently defines chunks.  We need to sort out referencing first (https://github.com/JacquesCarette/Drasil/issues/1029) before we can reasonably close this issue.
